lib/timeline/component/ItemSet.js

```diff
@@ -1388,12 +1388,14 @@ ItemSet.prototype._onDrag = function (event) {
       }
 
       var itemData = this._cloneItemData(props.item.data); // clone the data
-      if (props.item.editable === false && !me.options.editable.overrideItems) {
+      if (!props.item.editable.updateTime
+        && !props.item.editable.updateGroup
+        && !me.options.editable.overrideItems) {
         return;
       }
 
-      var updateTimeAllowed = me.options.editable.updateTime || (
-          props.item.editable === true && !me.options.editable.overrideItems);
+      var updateTimeAllowed = (me.options.editable.overrideItems && me.options.editable.updateTime) || (
```

Comment:
This overrides the previous default... what happens when:
`me.options.editable.overrideItems==false` and `me.options.editable.updateTime==true`?
updating time in this case should be allowed even if `props.item.editable.updateTime` is not defined (or false, as set in default)